conf/airframes/examples/ardrone2_rangesensors_gazebo.xml

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
```

Comment:
I do agree with this feedback, however it seems that in reality this is not a simple... Not only do I have to change the dynamics of the ardrone, I also have to change the values of imu, motor_mixing, servos (since the pocketdrone is in a plus configuration). 

The airframe will get very cluttered if I separate all of this in in the <target> under <firmware> tag, I wanted to do this in the <sections> or <servos> instead but the <target> tag does not exist there. Any ideas on how to make this more elegant?